test_runner/batch_others/test_wal_restore.py

```diff
@@ -24,15 +25,14 @@ def test_wal_restore(zenith_env_builder: ZenithEnvBuilder,
     env.zenith_cli.pageserver_stop()
     port = port_distributor.get_port()
     data_dir = os.path.join(test_output_dir, 'pgsql.restored')
-    restored = VanillaPostgres(data_dir, PgBin(test_output_dir), port)
-    subprocess.call([
-        'bash',
-        os.path.join(base_dir, 'zenith_utils/scripts/restore_from_wal.sh'),
-        os.path.join(pg_distrib_dir, 'bin'),
-        os.path.join(test_output_dir, 'repo/safekeepers/sk1/{}/*'.format(tenant_id)),
-        data_dir,
-        str(port)
-    ])
-    restored.start()
-    assert restored.safe_psql('select count(*) from t') == [(1000000, )]
-    restored.stop()
+    log_path = os.path.join(data_dir, "pg.log")
+    with VanillaPostgres(data_dir, PgBin(test_output_dir), port) as restored:
+        pg_bin.run_capture([
+            os.path.join(base_dir, 'zenith_utils/scripts/restore_from_wal.sh'),
+            os.path.join(pg_distrib_dir, 'bin'),
+            os.path.join(test_output_dir, 'repo/safekeepers/sk1/{}/*'.format(tenant_id)),
+            data_dir,
+            str(port)
+        ])
+        restored.start()
```

Comment:
Seems like this start() misses passing logging to `log_path`.

Reply:
Oops, fixed